### DISCUSSION TITLE
GCC for Windows

### DISCUSSION THREAD

**Discussion by shiroinekotfs** [shiroinekotfs/jupyter-cpp-kernel#11] [Q&A]
Hi all,

I'm here to take a look at the GCC for Windows.

I'm still curious about the best C++ compiler for Windows, just looking for the opportunity.

---

Moved from Issue https://github.com/takinekotfs/jupyter-cpp-kernel/issues/8

**Comment by shiroinekotfs** [comment]
@Welding-Torch I recommend you continue the discussion there.

**Reply by Welding-Torch** [reply]
Yes, most agreeable.
